src/docs/product/sentry-basics/guides/getting-started/index.mdx

```diff
@@ -1,128 +1,133 @@
-3. Mark an issue as **Resolved via Commit or PR** to stop notifications about similar errors in past releases and notify you upon regression.
-4. Link a Sentry issue to a new or existing issue in your integrated [Issue Tracking](#33-issue-tracking) solution
+1. **Suggest assignees** who can best resolve the error, based on the suspect commits and your [codeowners file](/product/issues/issue-owners/#code-owners).
+1. Mark an issue as **Resolved via Commit or PR** to stop notifications about similar errors in past releases and notify you of a regression.
+1. Link a Sentry issue to a new or existing issue in your integrated [issue tracking](#33-issue-tracking) solution.
+
+   ![Identify suspect commits and see suggested assignees for issues.](issue-integrations-01.png)
 
-   ![Integrate SCMs](07.png)
+   ![Link a Sentry issue to a new or existing issue in your issue tracking solution.](issue-integrations-02.png)
 
-Sentry has built-in integrations with:
+Sentry has built-in SCM integrations with:
 
+- [Azure DevOps](/product/integrations/source-code-mgmt/azure-devops/)
 - [GitHub](/product/integrations/source-code-mgmt/github/)
 - [GitLab](/product/integrations/source-code-mgmt/gitlab/)
 - [Bitbucket](/product/integrations/source-code-mgmt/bitbucket/)
 
-If you're using a different SCM provider or don't want Sentry to connect to your
-repository, check out our
-[releases documentation](/product/releases/setup/manual-setup-releases/)
-to learn how you can still enable most of these features (#1 - #3) by telling
-Sentry about your commit metadata.
+If you're using a different SCM provider or don't want Sentry to connect to your repository, check out our [Releases documentation](/product/releases/setup/manual-setup-releases/) to learn how you can still enable most of these features (#1 - #3) by telling Sentry about your commit metadata.
 
-### 3.3. Issue Tracking
+### Issue Tracking {#33-issue-tracking}
 
-Enabling the 2-way integration with your Issue Tracking solution allows you to create a new issue from within the Sentry issue page, or link it to an existing one. In addition to **GitHub**, **GitLab**, and **Bitbucket** Issues that are enabled with the respective SCM integration, Sentry also integrates with: [Azure DevOps](/product/integrations/source-code-mgmt/azure-devops/), [Clubhouse](/product/integrations/project-mgmt/clubhouse/), [JIRA](/product/integrations/project-mgmt/jira/), and others.
+Enabling an integration with your issue tracking solution allows you to create a new issue from within the **Issue Details** page in [sentry.io](https://sentry.io), or link to an existing one. In addition to GitHub, GitLab, and Bitbucket issues that are associated with the respective SCM integration, Sentry also integrates with [Azure DevOps](/product/integrations/source-code-mgmt/azure-devops/), [Clubhouse](/product/integrations/project-mgmt/clubhouse/), [Jira](/product/integrations/project-mgmt/jira/), and others.
```

Comment:
> sentry.io

I think you should make this just say Sentry. The external link icon isn't playing nice with the comma.

Reply:
> In addition to GitHub, GitLab, and Bitbucket issues that are associated with the respective SCM integration, Sentry also integrates with Azure DevOps, Clubhouse, Jira, and others.

I'm having a hard time parsing this sentence. Can we split it in two?

Reply:
> > sentry.io
> 
> I think you should make this just say Sentry. The external link icon isn't playing nice with the comma.

When we're talking about Sentry more generally, we use Sentry, but for the UI specifically, we use the linked url. And punctuation will inevitably follow outbound links sometimes. I don't think it's going to represent a blocker to comprehension.